about_Class.HostsManagement.help.txt

```diff
@@ -20,29 +20,28 @@ LONG DESCRIPTION
     - HostsEntryType (Enum)
       Represents the type of the HOSTS Entry line.
 
-
       Structure:
 
       [HostsFile]
         |--> [HostsEntry]
         |--> [HostsEntry]
         |--> [HostsEntry]
-                    |--> [HostsentryType]
```

Comment:
The TAB here was intentional, as it describes an indentation. A deeper level.
Please remove this.

Reply:
Okay Sure.